### ISSUE TITLE
Add hardware for checking output from Solar Panels

### ISSUE THREAD

**Issue by mage0r** [CCHS-Melbourne/ESPlant#13]
Will allow us to do some checking of light levels when there is not sufficient power to charge.  This would allow light detection inside for example, when the UV sensor would not work.

**Comment by projectgus** [comment]
Yeah, instead of reading the "maximum" of battery/solar we could use two ADC channels and read both.

With cuttable jumpers then they could be easily turned back into general purpose ADC channels as well, if people wanted more of those.

**Comment by mage0r** [comment]
Yeah, good call.  Lets do that.

**Comment by projectgus** [comment]
While we're noting problems with the current battery monitoring circuit, the voltage dividers should divide down to max of VRefInt (I think 1.1V) not 3.3V, because the 3.3V input is comparatively unstable. Reading against the internal reference will be more accurate.

**Comment by projectgus** [comment]
(and not droop if the battery gets very low and "3.3V" drops below 3.3V, etc.)

**Comment by marcmerlin** [comment]
Good suggestions on how to do this better than my cheap 10k x2 voltage divider :)
I'm currently using ADC2 for this as per the code that was just merged.
